### PR TITLE
fix error list colors in inverted-editor

### PR DESCRIPTION
**Pull request by jwunderl**
fix https://github.com/microsoft/pxt-minecraft/issues/1918, but there's a minor diff in styling for other editors I'll add a note on below; can maintain the same behavior but it'd require a bit of duplication / an extra less var defined for each editor

image

w/ highlight on error:

image

### CODE REVIEW

File: theme/themes/pxt/globals/site.variables [hunk 1]
```diff
@@ -358,6 +358,8 @@
 @errorListInnerHeight: (@errorListHeight - @errorListHeaderHeight);
 @errorListMessageHoverColor: rgba(0, 0, 0, 0.1);
 @errorListBorderColor: #e5e5e5;
-@errorListOdd: @homeScreenBackground;
```
Comment (jwunderl) [inline theme/themes/pxt/globals/site.variables:361]: This was assuming homeScreenBackground === monaco editor background, but that's only true if the editor specifically sets them the same; monaco colors are set like this https://github.com/microsoft/pxt-microbit/blob/master/pxtarget.json#L509 and they've been set to match for arcade & microbit but not minecraft (which has a light homescreen / dark editor).

As written this changes to just lighten / darken but it goes towards gray more than the current behavior with lesses darken; current arcade:

image

this change: 

image

I personally prefer the grayer one as it matches with the highlight color, but can add a variable to maintain hue like it currently does if preferred ~

File: theme/errorList.less [hunk 2]
```diff
@@ -117,4 +113,35 @@
...
+        color: @errorListInvertedTextColor;
+    }
+
+    .errorListInner {
+        .exceptionMessage {
+            color: @errorListInvertedTextColor;
+        }
+        .ui.selection.list {
+            color: @errorListInvertedTextColor;
+            cursor: pointer;
```
Comment (shakao) [inline theme/errorList.less:129]: do we need this in the non-inverted theme as well?

Comment (shakao) [reply theme/errorList.less:129]: same question for the border-radius/padding styles below

Comment (jwunderl) [reply theme/errorList.less]: Oops! I forgot to commit removing these, copy / paste error when grabbing the selectors.

**Pull request merged**